### DISCUSSION TITLE
Support for new Plex Smart Collections

### DISCUSSION THREAD

**Discussion by ExDominatus** [meisnate12/Plex-Meta-Manager#214] [Ideas]
Plex recently launched their new smart collections: https://support.plex.tv/articles/201273953-collections/. It basically solves a major issue with my collections which is allowing sort by the different orders that isn't Plex's dumb Alphabetical/Old>New. Being able to sort by Date Added, Release Date, and various Ratings is extremely powerful.

The issue is that these smart collections function more like smart playlists, in that you can't add items directly to them, only filter, atleast through the UI (ie. all German Horror movies)

I'm wondering if this feature has been released via API and if there are plans to support it or integrate it into PMM. The current workaround is to create a collection, filter by collection to create a smart collection, and sort that.

**Comment by meisnate12** [comment]
if [pkkid/python-plexapi](https://github.com/pkkid/python-plexapi) is updated to include smart collections ill add whatever i can from it
